examples/supertuxkart/Dockerfile

```diff
@@ -26,35 +24,22 @@ RUN go build -o wrapper .
 # compile stk
 FROM debian:buster as stk-builder
 
-RUN apt update && apt-get install -y build-essential cmake libbluetooth-dev \
-libcurl4-openssl-dev libenet-dev libfreetype6-dev libharfbuzz-dev libfribidi-dev \
-libgl1-mesa-dev libglew-dev libjpeg-dev libogg-dev libopenal-dev libpng-dev \
-libssl-dev libvorbis-dev libxrandr-dev libx11-dev nettle-dev pkg-config zlib1g-dev \
-git subversion
+RUN apt update && apt-get install -y  wget xz-utils
```

Comment:
Do we still need a seperate builder step, or can this all be done in a single image?

Since we're not compiling anymore, we probably don't need the builder step anymore I think.

(Although if we feel that we need a builder step, they should be on the same version of each OS)